README.md

```diff
@@ -3,7 +3,7 @@ API Usage
 =========
 
 To access Beey API, you need credentials first. Create an account at editor.beey.io. Your credentials are your e-mail and password that you use to log in Beey editor. The server URL is https://editor.beey.io/
-
+To find auth_token, access dev tools while on https://editor.beey.io/. While viewing project, you can find it under "network" tab, under user. Note, that if you log out from beey, the auth_token will change and will need to be changed in your project accordingly. 
```

Comment:
Nevím, pro jaký typu uživatele to je, ale mohlo by se hodit tam dopsat, že po otevření dev tools je potřeba refreshnout stránku (: